packages/contracts/package.json

```diff
@@ -1,38 +1,36 @@
 {
-    "private": true,
-    "name": "contracts",
-    "version": "2.1.48",
+    "name": "@0xproject/contracts",
+    "version": "2.0.0",
```

Comment:
Hm, what's the rationale here? Lerna is going to publish `patch` updates of this package whenever it's deps update. Don't think it's possible to have this version track the "contract" version. Rather we should maybe add a second key called `"0xVersion: "2.0.0"`

Reply:
I was thinking it would be cleaner to start at `2.0.0` and it shouldn't matter since this package has never been published before.

That said, you're right that in the long-term this version will probably not track the "contract" version exactly. I changed my mind and I think it's probably better to keep the same version number we had before. I'll change it back to `2.1.48`.